tests/integration/components/task-card-test.js

```diff
@@ -293,3 +300,48 @@ test('assignment dropdown does not render if user has no ability', function(asse
     assert.equal(user.icon.url, 'test.png');
   });
 });
+
+test('the selected-item component is visable when a task has a user assigned ', function(assert) {
+  assert.expect(5);
+
+  let task = { id: 'task' };
+  let user = { id: 'user', username: 'testuser', photoThumbUrl: 'test.png' };
+  let users = [user];
+  let taskUser = user;
+
+  setProperties(this, { task, users, taskUser });
+
+  renderPage();
+  assert.equal(page.selectedItem.selectedIcon.src, taskUser.photoThumbUrl, 'the selected user photo icon renders');
+  assert.notOk(page.tooltip, 'there is no tooltip because mouse is not hovering.');
+  assertTooltipNotRendered(assert);
+
+  page.selectedItem.mouseenter();
+
+  assertTooltipVisible(assert);
+  assert.equal(page.selectedItem.tooltip.text, `Assigned to ${taskUser.username}`, 'the tooltip renders the correct text.');
+});
+
+test('the unselected-item component is visable when a task has a user assigned ', function(assert) {
+  assert.expect(4);
+  let task = { userTask: { isLoading: false }, id: 'task' };
+  let user = { id: 'user', username: 'testuser' };
+  let users = [user];
+
+  setProperties(this, { task, users });
+
+  renderPage();
+  assert.ok(page.unselectedItem.unselectedIcon.isVisible, 'the unselected user icon renders.');
```

Comment:
You're one step closer to what I proposed above, but you could just as easily test for 

```Javascript
assert.ok(page.unselectedItem.isVisible, 'The unselected item component renders on the card.');
```